### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ktras**
Based on conversation with @rouson about comments that @bonachea had made, change ABI to API in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Caffeine
 
 **CoArray Fortran Framework of Efficient Interfaces to Network Environments**
 
-Caffeine is a parallel runtime library that aims to support Fortran compilers with a programming-model-agnostic application binary interface (ABI) to various communication libraries.  Current work is on supporting the ABI with the [GASNet-EX] exascale-ready networking middleware.  Future plans include support for an alternative Message Passing Interface ([MPI]) back end.
+Caffeine is a parallel runtime library that aims to support Fortran compilers with a programming-model-agnostic application programming interface (API) to various communication libraries.  Current work is on supporting the API with the [GASNet-EX] exascale-ready networking middleware.  Future plans include support for an alternative Message Passing Interface ([MPI]) back end.
```
Comment (rouson) [inline README.md:6]: @ktras I think we've now moved on from "API" to "PRIF" or just "interface".   Please adjust accordingly.

**Review by everythingfunctional — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (ktras) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ Caffeine
 
 **CoArray Fortran Framework of Efficient Interfaces to Network Environments**
 
-Caffeine is a parallel runtime library that aims to support Fortran compilers with a programming-model-agnostic application binary interface (ABI) to various communication libraries.  Current work is on supporting the ABI with the [GASNet-EX] exascale-ready networking middleware.  Future plans include support for an alternative Message Passing Interface ([MPI]) back end.
+Caffeine is a parallel runtime library that aims to support Fortran compilers with a programming-model-agnostic application interface to various communication libraries.  Current work is on supporting the Parallel Runtime Interface for Fortran (PRIF) with the [GASNet-EX] exascale-ready networking middleware.  Future plans include support for an alternative Message Passing Interface ([MPI]) back end.
```
Comment (bonachea) [inline README.md:6]: Please make the PRIF text a hyperlink to the spec : https://doi.org/10.25344/S4DG6S

Please also add the PRIF spec at the bottom of the publication list, with complete citation:

Damian Rouson, Brad Richardson, Dan Bonachea, Katherine Rasmussen, "Parallel Runtime Interface for Fortran (PRIF) Design Document, Revision 0.2", Lawrence Berkeley National Laboratory Technical Report (LBNL-2001563), Dec 20 2023,
https://doi.org/10.25344/S4DG6S

**Review by bonachea — CHANGES_REQUESTED** [review]

**Pull request merged**